extensions/data-operation/data-operation-actions/pom.xml

```
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.apache.unomi</groupId>
+        <artifactId>data-operation</artifactId>
+        <version>1.5.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>data-operation-actions</artifactId>
+
+    <name>Apache Unomi :: Extensions :: Kafka Buffer ActionActions</name>
```

Small typo found :) ActionActions
```suggestion
    <name>Apache Unomi :: Extensions :: Kafka Buffer Action</name>
```